core/block_svg.js

```diff
@@ -150,9 +150,11 @@ Blockly.BlockSvg.prototype.select = function() {
     return;
   }
   if (Blockly.selected == this) {
-    return;
+    var oldId = this.id;
```

Comment:
it is not a good style to declare variables twice (and is only working in javascript)